### PR TITLE
Add a section to Blog/embertimes 92

### PR DESCRIPTION
**Pull request by lisaychuang**
Add a section to Ember Times #46 for my post on volunteering at EmberConf.

### CODE REVIEW

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 1]
```diff
@@ -12,10 +12,16 @@ responsive: true
...
+## [3 Reasons To Volunteer At a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
+
+[@lisaychuang](https://twitter.com/lisaychuang) shares her experience from this year’s EmberConf, and encourages newcomers in any tech community to attend a tech conference (and possibly volunteer!)   
+
+As a volunteer, you can get:
+* A warm welcome to the community
+* Behind the scene access
+* A chill home base during the conference
+
+Curious to learn more? Check out her post on Medium:[3 reasons why you should volunteer at a tech conference](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8).
```
Comment (amyrlam) [inline source/2019-04-05-the-ember-times-issue-92.md]: The suggestions UI seems to be broken for me rn 😢, but add a space after `Medium:`

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 2]
```diff
@@ -12,10 +12,16 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [3 Reasons To Volunteer At a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
```
Comment (amyrlam) [inline source/2019-04-05-the-ember-times-issue-92.md]: Love the 💪

Comment (amyrlam) [reply source/2019-04-05-the-ember-times-issue-92.md]: Change to `3 Reasons to Volunteer at a Tech Conference 💪🏼`, see https://titlecase.com/. I will also update CONTRIBUTING, this is a better link than the one we have. See: https://github.com/ember-learn/ember-blog/pull/50

File: source/2019-04-05-the-ember-times-issue-92.md (hunk 1 above)
Comment (jayjayjpg) [reply source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
Curious to learn more? Check out her post on Medium: [3 reasons why you should volunteer at a tech conference](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8).
```

File: source/2019-04-05-the-ember-times-issue-92.md (hunk 2 above)
Comment (jayjayjpg) [reply source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
## [3 Reasons to Volunteer at a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
```

**Comment by lisaychuang** [comment]
Thank you for the suggested changes!

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for this :sparkles: writeup this week @lisaychuang!

**Comment by lisaychuang** [comment]
Awww thanks @jessica-jordan !

**Pull request merged**